Library/Homebrew/utils/livecheck.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Livecheck
+  def livecheck_formula_response(name)
+    ohai "Checking livecheck formula : #{name}"
+
+    response = Utils.popen_read("brew", "livecheck", name, "--quiet").chomp
```

Comment:
Hi @MikeMcQuaid 

Today in Discord we discussed with you the best way to query livecheck from within our code, and also updated the code to use `Utils.popen_read` as suggested and seen above.  Our understanding from your comments was that we would need to use `brew livecheck` in order to livecheck a formula.  We have run into a bit of a blocker I'm hoping you can advise on.

Line 7 above, `Utils.popen_read("brew", "livecheck", name, "--quiet").chomp` works just fine if we run this `livecheck.rb` file.  Example output if we display the response:
```
ruby Homebrew/utils/livecheck.rb
# . . .
"openclonk : 7.0 ==> 8.1"
# . . 
```
However when this code is run as a consequence the `brew bump` command @BaffourAdu is working on implementing (in `dev-cmd/bump.rb`), an empty string is returned.  

#### What we've tried:
- We tried replacing "livecheck" with another brew command-- `Utils.popen_read("brew", "release-notes")`, which also returns an empty string.
- We tried replacing our arguments with something unrelated to brew-- `Utils.popen_read("git", "status")` and `Utils.popen_read('ls')`, and we *do* see the expected output.
- Paired with Matthew Bunday to see if he had any ideas on how to address this.
- Looked through the existing `Homebrew/brew` codebase and couldn't find any examples to emulate in which a `brew` command is being executed like this.

Given that we see the expected output when the arguments passed to `Utils.popen_read` are not a `brew` command, we suspect there is some difficulty with trying to livecheck the formulae via `brew livecheck` as a part of the execution of our `brew bump` command.

I hope this was clear, & we appreciate any guidance you can provide on this issue.  Thank you!

Reply:
Repeating from Discord just so it's here: try `Utils.popen_read(HOMEBREW_BREW_FILE, "livecheck", name, "--quiet")`, that should work.